kubectl-pg/glide.yaml

```diff
@@ -0,0 +1,26 @@
+package: github.com/zalando/postgres-operator/kubectl-pg
+import:
+- package: github.com/spf13/cobra
+- package: github.com/zalando-incubator/postgres-operator
+  subpackages:
+  - pkg/apis/acid.zalan.do/v1
+  - pkgs/apis/acid.zalan.do/v1
```

Comment:
remove package github.com/zalando-incubator/postgres-operator and subpackages

Reply:
I guess it should be simply `zalando` without the `incubator` part